enterprise/api/v1/views/pending_enterprise_customer_admin_user.py

```diff
@@ -0,0 +1,30 @@
+"""
+Views for the ``pending-enterprise-customer-admin-user`` API endpoint.
+"""
+
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import filters, permissions
+
+from enterprise import models
+from enterprise.api.v1 import serializers
+from enterprise.api.v1.views.base_views import EnterpriseReadWriteModelViewSet
+from enterprise.logging import getEnterpriseLogger
+
+LOGGER = getEnterpriseLogger(__name__)
+
+
+class PendingEnterpriseCustomerAdminUserViewSet(EnterpriseReadWriteModelViewSet):
```

Comment:
Question: How do we ensure that a user who is already a part of a customer does not get added to the PendingEnterpriseCustomerAdmin list?